src/components/index.ts

```diff
@@ -109,6 +110,7 @@ export * from './MessageInput/UploadProgressIndicator';
 export * from './MessageList/DateHeader';
 export * from './MessageList/hooks/useMessageList';
 export * from './MessageList/hooks/useTypingString';
+export * from './MessageList/InlineDateIndicator';
```

Comment:
Can we call this `InlineDateSeparator`? Sounds more like separator than indicator

Reply:
Also let's make it customisable through Channel component. Quite a common use-case

Reply:
Can do on the name, and, yup that was my plan for it.

Reply:
ahaha sorry, I jumped on draft!! 😬